clients/vscode/src/chat/ChatViewProvider.ts

```diff
@@ -404,6 +425,11 @@ export class ChatViewProvider implements WebviewViewProvider {
                 window.addEventListener("message", (event) => {
                   if (!chatIframe) return
                   if (event.data) {
+
+                    if (event.data.action === 'keydown') {
+                      vscode.postMessage(event.data);
+                      return;
+                    }
```

Comment:
instead of post message to vscode, add it to client api:
https://demo.tabbyml.com/files/github/TabbyML/tabby/-/blob/83b6748b768c9061f6b17d9fc94f7a8e9194f22d/clients/tabby-chat-panel/src/index.ts#L44

e.g `focusEditor`